contents/blog/pricing-lessons.md

```diff
@@ -0,0 +1,96 @@
+
+**Encourage users to upgrade**, since you need people to do exactly that! We encourage people to put in their credit card - by offering our paid features starting at $0 (with a generous amount of free usage) if they do so. This lets users evaluate the paid product very easily, and it lets them upgrade with no friction if they're heavy users. This exchange feels fair to us - it helps us figure out who is serious versus playing around. Our numbers reflect that users feel the same way - over half our users enter their card details.
+
+**No minimum spend**, so paying users can start small if they want to. This means a customer can gradually get more committed to you. If customers ask for trials, you can get them straight into a paying relationship "just keep the volume low", with a seamless way to upgrade - use the product more. That makes it easier for customers to avoid some complex processes around pilots/proofs of concept, and it stops a jarring discussion around why someone is going from $0/month to $10k/month. Don't let your pricing generate a stressful decision point.
+
+**No annual contracts**, unless a customer requests one. This cuts the risk down for customers, reducing the need for intensive evaluation periods. Ironically, the less commitment you ask for, the more you may get. Of course, some customers _will_ want you to do this, to lock in pricing, but don't default to this.
+
+**Be transparent about pricing** so users don't have to 'talk to sales'. Is the best way to close a customer to signal to them that you're going to charge them as much as you can get away with? Many people think so. Disclaimer: we started non-transparent so we could speak to some users to figure out the right model first (and we still do this - see our non-transparent enterprise pricing)... but we make it transparent as soon as we've reasonable confidence.
+
+**Be self serve** so users can start paying you to solve a problem they have _right now_. Defaulting to _not_ using a sales team, and providing transarent pricing, means you can get people in the moment.
+
+## But don't lose the ability to do big contracts
+
+* We differentiated on features
+* Our usage grows with our customers' products. If we charged a flat fee, we couldn't do this so easily.
+
+Just because it's easy to get started, doesn't mean your product has to be cheap.
+
+Some companies with usage pricing get criticized for the escalating costs at scale. If it takes just 1 extra engineer to self build in a large company, you theoretically shold be able to charge >$100K annually. If you've competition, it depends what they charge. 
+
+## You don't have to do all these things immediately
+
+Early on, optimize for learning. Later on, optimize for the best solution for your users.
+
+**We didn't start with a $0 minimum**. Last summer, we worked on finding just five reference customers for our paid product (we now have X customers).
```

Comment:
Is the (we now have X customers) a reference to something or is this a number that needs filling in?